php-fpm/recipes/configure.rb

```diff
@@ -69,6 +69,23 @@
   end
 end
 
+#
+# An attempt to make sure that the php prefix given is the prefix
+# chosen for php-cli at the end of an install
+php_package_prefix = node['php']['ppa']['package_prefix'].gsub(/php/, '')
```

Comment:
I used my own because in my test runs the php version came up empty.  Something about the difference in gsub usage.